markdown/ch4.markdown

```diff
@@ -291,7 +290,7 @@ freqs, times, Sx = signal.spectrogram(audio, fs=rate, window='hanning',
                                       detrend=False, scaling='spectrum')
 
 ## Plot using:
-# plt.pcolormesh(times, freqs, 10 * np.log10(Sx), cmap='viridis');
+# plt.pcolormesh(times, freqs, 10 * np.log10(Sx));
```

Comment:
That's why I added the comment, because you can't just do an imshow with scipy.  But, OK, let me uncomment that.